source/Nevermore.Tests/QueryBuilderFixture/QueryBuilderFixture.cs

```diff
@@ -1887,6 +1887,51 @@ public void ShouldGenerateUniqueParameterNamesInJoin()
 
             actual.Should().BeEquivalentTo(expected);
         }
+        
+        [Test]
+        public void ShouldGenerateStaticJoinAliasNamesInJoin()
```

Comment:
This test is a copy of `ShouldGenerateUniqueParameterNamesInJoin` above it, but with static aliases